lib/team.rb

```diff
@@ -1,6 +1,9 @@
 class Team
+  attr_reader :team_id, :team_name
+
   def initialize(details)
-    @team_id = details[:team_id]
+    @team_id = details[:team_id].to_i
+    @team_name = details[:teamname]
   end
```

Comment:
Same thing with @team_id. We will need to figure out a consistent data type that works for all of us